bgapi/module.py

```diff
@@ -460,6 +497,8 @@ def disable_scan(self, scan_interval = 0x4B, scan_window = 0x32):
 
     def connect(self, target, timeout=5, conn_interval_min=0x20, conn_interval_max=0x30, connection_timeout=100, latency=0):
         self.start_procedure(CONNECT)
+        print target.sender, type(target.sender)
```

Comment:
print commands don't work well when doing headless projects on servers.  They can spew into random log places.
This file should have "logger = logging.getLogger('bgapi')" at the top just under the imports.  Then use "logger.log(logging.INFO, "%s %s" % (target.sender, type(target.sender))" here.  This also has the bonus of adding a timestamp and the serial port pre-pended to the log message.  It helps a lot to have these for multiple-dongle projects, though it's a unique use case.
If you don't want to mess with logging, just comment these out and leave them here.